sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -649,6 +648,16 @@ class VersionsSuite extends QueryTest with SQLTestUtils with TestHiveSingleton w
       }
     }
 
+    test(s"$version: create table should success to test HiveClientImpl.toHiveTable compatible") {
+      withTable("t", "t1") {
+        import spark.implicits._
+        Seq("1").toDF("a").write.saveAsTable("t")
+        checkAnswer(spark.table("t"), Row("1") :: Nil)
+
+        spark.sql("create table t1 as select 2 as a")
+        checkAnswer(spark.table("t1"), Row(2) :: Nil)
```

Comment:
Table `t` and `t1` are managed tables. Could you also verify whether the location are expected?